ui/client/components/graph/node-modal/editors/expression/FixedValuesEditor.tsx

```diff
@@ -52,18 +52,17 @@ export default class FixedValuesEditor extends React.Component<Props> {
     const option = this.currentOption()
 
     return (
-      <React.Fragment>
+      <div className={`node-value-select ${className}`}>
```

Comment:
is it necessary to fix editor behaviour?

Reply:
yep, otherwise was some problems with layout of rendered select box. it was a regression bug introduced during adding support for fixed value editor in process properties - after my change, in both situations it looks ok.